emu/templates/Dockerfile.emulator

```diff
@@ -53,18 +53,96 @@ EXPOSE 5555
 # This is the gRPC port, also useful, we don't want ADB to incorrectly identify this.
 EXPOSE 8554
 
-WORKDIR /android/sdk
+WORKDIR /root
 
 # You will need to make use of the grpc snapshot/webrtc functionality to actually interact with
 # the emulator.
-CMD ["/android/sdk/launch-emulator.sh"]
+# CMD ["/android/sdk/launch-emulator.sh"]
 
 # Note we should use gRPC status endpoint to check for health once the canary release is out.
+# HEALTHCHECK --interval=30s \
```

Comment:
this dockerfile will be merged to the one in infra-scripts ?

Reply:
this dockerfile will be generated with both appium+emulator combine inside src when we build the dockerfile of infra-scripts